components/moonpay/buildflags/buildflags.gni

```diff
@@ -0,0 +1,9 @@
+import("//build/config/features.gni")
+
+declare_args() {
+  moonpay_enabled = is_mac || is_linux || is_win
+}
+
+declare_args() {
+  bitcoin_dot_com_enabled = moonpay_enabled
```

Comment:
There shouldn't be 2 `declare_args`
Wondering if we need both `moonpay_enabled` and `bitcoin_dot_com_enabled`?

Reply:
merged these in to one `declare_args()` call for now